### ISSUE TITLE
Using multiple API keys for Spotter class

### ISSUE THREAD

**Issue by benjwadams** [sofarocean/sofar-api-client-python#7]
If it is desired to use mutliple API keys with the Spotter class, only the first assigned API key from the environment variable `WF_API_TOKEN` get used at import time.  Thus, if a user wants to use a different spotter with a different API key, they will have to do something like:

```python
import os
import importlib
import pysofar
import pysofar.spotter
# this is a hack, requires setting an environment variable
# and reloading modules any time the API key potentially changed
os.environ["WF_API_TOKEN"] = api_key
importlib.reload(pysofar)
importlib.reload(pysofar.spotter)
spotter = pysofar.spotter.Spotter(spotter_id, "")
```

It would be preferable to be able to do this by having a usable argument from the `Spotter` constructor or similar.

**Comment by CyberAngler93** [comment]
I had the same issues, I am working on showcasing some work using jupyter notebooks and outside of a hacky approach it would be nice to set your own `custom_token`. I found that there is a constructor param that checks if the flag exists. 
After looking into this code, I believe that the issue lies within the `__init__` as it does not properly handle the custom_token flag that can be passed on the API creation. The custom_token is not documented in the docs. I have opened an [issue](https://github.com/sofarocean/sofar-api-client-python/issues/8)  and submitted a [PR](https://github.com/sofarocean/sofar-api-client-python/pull/9)  for a suggested fix.
